### PR TITLE
tp-qemu: fix ppc platform no io_exits and irq_injections issue

### PR DESCRIPTION
**Pull request by mazhang**
ID: 1315548 1315167

Signed-off-by: Maosheng Zhang <mazhang@redhat.com>

### CODE REVIEW

**Comment by Zhengtong** [comment]
Acked-by: Zhengtong Liu <zhengtli@redhat.com>

File: generic/tests/netperf.py [hunk 1]
```diff
@@ -521,21 +533,27 @@ def get_state():
...
+
         return state_list
 
     def netperf_thread(i, numa_enable, client_s, timeout):
         cmd = ""
         fname = "/tmp/netperf.%s.nf" % pid
         if numa_enable:
-            output = ssh_cmd(client_s, "numactl --hardware")
-            n = int(re.findall(r"available: (\d+) nodes", output)[0]) - 1
+            output = ssh_cmd(client_s, "numactl --show|grep nodebind")
```
Comment (xutian) [inline generic/tests/netperf.py]: It looks use command 'numactl --hardware' to find a available node looks more reasonable.

**Comment by xutian** [comment]
@mazhang  Please make a hotfix please !!!

**Comment by mazhang** [comment]
@xutian fixed, please check it again.

**Comment by MiriamDeng** [comment]
For the hotfix,it can not help to get corret number.
If node is 17,only get "7".

File: generic/tests/netperf.py [hunk 2]
```diff
@@ -535,7 +553,7 @@ def netperf_thread(i, numa_enable, client_s, timeout):
         fname = "/tmp/netperf.%s.nf" % pid
         if numa_enable:
             output = ssh_cmd(client_s, "numactl --hardware")
-            n = int(re.findall(r"available: (\d+) nodes", output)[0]) - 1
+            n = int(re.findall(r"node (\d+) cpus:", output)[-1])
```
Comment (xutian) [inline generic/tests/netperf.py]: no need to convert  it to int , because you use it with '%s'

**Comment by MiriamDeng** [comment]
Acked-by: Min Deng  mdeng@redhat.com
The latest hot fix is applied correctly.

**Pull request merged**